_docs/2.0.0/usage/spark-jobs-quick-start.md

```diff
@@ -7,8 +7,97 @@ categories:
+- `report_date` represents the date on which the data landed in the HDFS (in raw) and also the suffix part of the data output path. So if in Menas raw was specified as `/path/on/hdfs/raw` and the input parameter `report_date` as `2020-12-24` then the path where standardization will look for input files will be `/path/on/hdfs/raw/2020/12/24`. For the final part we are missing the report versions.
+- `report_version` is the final part of the path on HDSF. With `report_date` we finished with `/path/on/hdfs/raw/2020/12/24/v<report_version>`. This is the location where standardization will look for raw data.
+- `raw-format` and its specifics. Raw format tells the standardization which format the data is in on the HDFS and what are its specifics. CSV might have a header, XML has a row-tag, etc. Here in the example, we use the `row-tag`. For more options for different types and run parameters see our [run documentation](https://absaoss.github.io/enceladus/docs/2.0.0/usage/run) or just run `--help`
+
+
+### Running Conformance
+
+```shell
+<spark home>/spark-submit \
+--num-executors 2 \
+--executor-memory 2G \
+--master yarn \
+--deploy-mode cluster \
+--driver-cores 2 \
+--driver-memory 2G \
+--class za.co.absa.enceladus.conformance.DynamicConformanceJob \
+spark-jobs_<build_version>.jar \
+--menas-auth-keytab <path_to_keytab_file> \
+--dataset-name <dataset_name> \
+--dataset-version <dataset_version> \
+--report-date <report_date> \
+--report-version <data_run_version>
+```
+
+Here, nothing new is added for the quick run. Of course, there might be special options which are all documented in the [run documentation](https://absaoss.github.io/enceladus/docs/2.0.0/usage/run)
+
+## Running with helper scripts
+
+If your local DevOps/SysAdmin set up helper scripts for you, then it is even easier. You can omit all the spark options if sensible defaults are provided or Dynamic Resource Allocation is enabled. For more about this ask the people who set up your environment.
+
+When scripts are properly set up, then only a few parameters need to specified.
+
+The basic command to run Standardization becomes:
+
+```shell
+<path to scripts>/run_standardization.sh \
+--menas-auth-keytab <path_to_keytab_file> \
+--dataset-name <dataset_name> \
+--dataset-version <dataset_version> \
+--report-date <report_date> \
+--report-version <data_run_version> \
+--raw-format <data_format> \
+--row-tag <tag>
+```
+
+The basic command to run Conformance becomes:
+
+```shell
+<path to scripts>/run_conformance.sh \
+--deploy-mode <client/cluster> \
+--menas-auth-keytab <path_to_keytab_file> \
+--dataset-name <dataset_name> \
+--dataset-version <dataset_version> \
+--report-date <report_date> \
+--report-version <data_run_version>
+```
+
+For more options and arguments check the [run documentations](https://absaoss.github.io/enceladus/docs/2.0.0/usage/run)
```

Comment:
You left "documentations" here, but it is just a detail.

Reply:
Not just a detail. I did not see the comment under the suggestions. Fixed. Thank you